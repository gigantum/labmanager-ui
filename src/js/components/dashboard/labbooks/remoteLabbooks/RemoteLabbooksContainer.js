import Labbooks from '../Labbooks';
import {
  createFragmentContainer,
  graphql,
} from 'react-relay';

export default createFragmentContainer(
  Labbooks,
  graphql`
    fragment RemoteLabbooksContainer_labbookList on LabbookQuery{
      labbookList{
        id
        ...RemoteLabbooks_remoteLabbooks
      }
    }
  `,
);
