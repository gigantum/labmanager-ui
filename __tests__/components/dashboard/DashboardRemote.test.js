
      import React from 'react'
      import renderer from 'react-test-renderer';
      import history from 'JS/history'
      import {mount} from 'enzyme'
      import RemoteLabbooksContainer from 'Components/dashboard/labbooks/remoteLabbooks/RemoteLabbooksContainer';
      import {Provider} from 'react-redux'
      import store from "JS/redux/store"
      import json from './__relaydata__/DashboardRemote.json'

      import relayTestingUtils from 'relay-testing-utils'

      history.location.pathname = 'hostname/labbooks/cloud'

      const fixtures = {
        auth: ()=>{

        },
        labbookList: json.data,
        remoteLabbooks: json.data.labbookList,
        history: history,
        refetchSort: ()=>{

        }
      }

      test('Test DashboardRemote', () => {

        const wrapper = renderer.create(

           relayTestingUtils.relayWrap(
            <Provider store={store}>
              <RemoteLabbooksContainer {...fixtures} />
            </Provider>
           , {}, json.data)

        );


        const tree = wrapper.toJSON()

        expect(tree).toMatchSnapshot()

      })
