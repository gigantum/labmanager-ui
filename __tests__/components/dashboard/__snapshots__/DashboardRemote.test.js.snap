// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test DashboardRemote 1`] = `
<div
  className="Labbooks"
>
  <div />
  <div
    className="Labbooks__title-bar"
  >
    <h6
      className="Labbooks__username"
    >
      cbutler
    </h6>
    <h2
      className="Labbooks__title"
      onClick={[Function]}
    >
      Projects
    </h2>
  </div>
  <div
    className="Labbooks__menu  mui-container flex-0-0-auto"
  >
    <ul
      className="Labbooks__nav  flex flex--row"
    >
      <li
        className="Labbooks__nav-item--0"
      >
        <a
          onClick={[Function]}
        >
          Local
        </a>
      </li>
      <li
        className="Labbooks__nav-item--1 selected"
      >
        <a
          onClick={[Function]}
        >
          Cloud
        </a>
      </li>
      <hr
        className="Labbooks__navigation-slider Labbooks__navigation-slider--1"
      />
      <div
        className="hidden cloudLocal"
      >
        <div
          className="ToolTip__toggle cloudLocal"
          onClick={[Function]}
        >
          <div
            className="ToolTip__glow-container"
          >
            <div
              className="ToolTip__glow-ring-outer"
            >
              <div
                className="ToolTip__glow-ring-inner"
              />
            </div>
          </div>
        </div>
        <div
          className="hidden cloudLocal"
        />
        <div
          className="hidden cloudLocal"
        >
          This allows you to switch between Local and Cloud Project listings. Both Projects you have created and Projects that have been shared with you will be visible.
        </div>
      </div>
    </ul>
  </div>
  <div
    className="Labbooks__subheader"
  >
    <div
      className="Labbooks__search-container"
    >
      <input
        className="Labbooks__search no--margin"
        defaultValue=""
        onFocus={[Function]}
        onKeyUp={[Function]}
        placeholder="Filter Projects by name or description"
        type="text"
      />
    </div>
    <div
      className="Labbooks__filter"
    >
      Filter by:
      <span
        className="Labbooks__filter-collapsed"
        onClick={[Function]}
      >
        All
      </span>
      <ul
        className="hidden"
      >
        <li
          className="Labbooks__filter-item"
          onClick={[Function]}
        >
          All 
          ✓ 
        </li>
        <li
          className="Labbooks__filter-item"
          onClick={[Function]}
        >
          My Projects 
          
        </li>
        <li
          className="Labbooks__filter-item"
          onClick={[Function]}
        >
          Shared with me 
          
        </li>
      </ul>
    </div>
    <div
      className="Labbooks__sort"
    >
      Sort by:
      <span
        className="Labbooks__sort-collapsed"
        onClick={[Function]}
      >
        Modified Date (Newest)
      </span>
      <ul
        className="hidden"
      >
        <li
          className="Labbooks__sort-item"
          onClick={[Function]}
        >
          Modified Date (Newest) 
          ✓ 
        </li>
        <li
          className="Labbooks__sort-item"
          onClick={[Function]}
        >
          Modified Date (Oldest) 
          
        </li>
        <li
          className="Labbooks__sort-item"
          onClick={[Function]}
        >
          Creation Date (Newest) 
          
        </li>
        <li
          className="Labbooks__sort-item"
          onClick={[Function]}
        >
          Creation Date (Oldest) 
          
        </li>
        <li
          className="Labbooks__sort-item"
          onClick={[Function]}
        >
          A-Z 
          
        </li>
        <li
          className="Labbooks__sort-item"
          onClick={[Function]}
        >
          Z-A 
          
        </li>
      </ul>
    </div>
  </div>
  <div
    className="LocalLabbooks__labbooks"
  >
    <div
      className="LocalLabbooks__sizer grid"
    >
      <div
        className="RemoteLabbooks__panel column-4-span-3 flex flex--column justify--space-between"
      >
        <div
          className="RemoteLabbooks__icon-row"
        >
          <button
            className="RemoteLabbooks__cloud-download"
            disabled={false}
            onClick={[Function]}
          />
          <button
            className="RemoteLabbooks__cloud-delete--disabled"
            disabled={false}
            onClick={[Function]}
          />
        </div>
        <div
          className="RemoteLabbooks__text-row"
        >
          <div
            className="RemoteLabbooks__title-row"
          >
            <h6
              className="RemoteLabbooks__panel-title"
            >
              <span
                className={undefined}
              >
                <span
                  className=""
                  style={undefined}
                >
                  update-test
                </span>
              </span>
            </h6>
          </div>
          <p
            className="RemoteLabbooks__owner"
          >
            Created by dmk
          </p>
          <p
            className="RemoteLabbooks__description"
          >
            <span
              className={undefined}
            />
          </p>
        </div>
        <div
          className="Tooltip RemoteLabbooks__undefined"
          data-tooltip="undefined"
        />
      </div>
      <div
        className="RemoteLabbooks__panel column-4-span-3 flex flex--column justify--space-between"
      >
        <div
          className="RemoteLabbooks__icon-row"
        >
          <button
            className="RemoteLabbooks__cloud-download--exists"
            disabled={true}
          />
          <button
            className="RemoteLabbooks__cloud-delete--disabled"
            disabled={false}
            onClick={[Function]}
          />
        </div>
        <div
          className="RemoteLabbooks__text-row"
        >
          <div
            className="RemoteLabbooks__title-row"
          >
            <h6
              className="RemoteLabbooks__panel-title"
            >
              <span
                className={undefined}
              >
                <span
                  className=""
                  style={undefined}
                >
                  trump-tweets
                </span>
              </span>
            </h6>
          </div>
          <p
            className="RemoteLabbooks__owner"
          >
            Created by dmk
          </p>
          <p
            className="RemoteLabbooks__description"
          >
            <span
              className={undefined}
            />
          </p>
        </div>
        <div
          className="Tooltip RemoteLabbooks__undefined"
          data-tooltip="undefined"
        />
      </div>
      <div
        className="RemoteLabbooks__panel column-4-span-3 flex flex--column justify--space-between"
      >
        <div
          className="RemoteLabbooks__icon-row"
        >
          <button
            className="RemoteLabbooks__cloud-download"
            disabled={false}
            onClick={[Function]}
          />
          <button
            className="RemoteLabbooks__cloud-delete--disabled"
            disabled={false}
            onClick={[Function]}
          />
        </div>
        <div
          className="RemoteLabbooks__text-row"
        >
          <div
            className="RemoteLabbooks__title-row"
          >
            <h6
              className="RemoteLabbooks__panel-title"
            >
              <span
                className={undefined}
              >
                <span
                  className=""
                  style={undefined}
                >
                  tpub
                </span>
              </span>
            </h6>
          </div>
          <p
            className="RemoteLabbooks__owner"
          >
            Created by bill2
          </p>
          <p
            className="RemoteLabbooks__description"
          >
            <span
              className={undefined}
            />
          </p>
        </div>
        <div
          className="Tooltip RemoteLabbooks__undefined"
          data-tooltip="undefined"
        />
      </div>
      <div
        className="RemoteLabbooks__panel column-4-span-3 flex flex--column justify--space-between"
      >
        <div
          className="RemoteLabbooks__icon-row"
        >
          <button
            className="RemoteLabbooks__cloud-download--exists"
            disabled={true}
          />
          <button
            className="RemoteLabbooks__cloud-delete--disabled"
            disabled={false}
            onClick={[Function]}
          />
        </div>
        <div
          className="RemoteLabbooks__text-row"
        >
          <div
            className="RemoteLabbooks__title-row"
          >
            <h6
              className="RemoteLabbooks__panel-title"
            >
              <span
                className={undefined}
              >
                <span
                  className=""
                  style={undefined}
                >
                  test-lb
                </span>
              </span>
            </h6>
          </div>
          <p
            className="RemoteLabbooks__owner"
          >
            Created by dmk
          </p>
          <p
            className="RemoteLabbooks__description"
          >
            <span
              className={undefined}
            />
          </p>
        </div>
        <div
          className="Tooltip RemoteLabbooks__undefined"
          data-tooltip="undefined"
        />
      </div>
      <div
        className="RemoteLabbooks__panel column-4-span-3 flex flex--column justify--space-between"
      >
        <div
          className="RemoteLabbooks__icon-row"
        >
          <button
            className="RemoteLabbooks__cloud-download--exists"
            disabled={true}
          />
          <button
            className="RemoteLabbooks__cloud-delete--disabled"
            disabled={false}
            onClick={[Function]}
          />
        </div>
        <div
          className="RemoteLabbooks__text-row"
        >
          <div
            className="RemoteLabbooks__title-row"
          >
            <h6
              className="RemoteLabbooks__panel-title"
            >
              <span
                className={undefined}
              >
                <span
                  className=""
                  style={undefined}
                >
                  testing-notifications
                </span>
              </span>
            </h6>
          </div>
          <p
            className="RemoteLabbooks__owner"
          >
            Created by dmk
          </p>
          <p
            className="RemoteLabbooks__description"
          >
            <span
              className={undefined}
            />
          </p>
        </div>
        <div
          className="Tooltip RemoteLabbooks__undefined"
          data-tooltip="undefined"
        />
      </div>
      <div
        className="RemoteLabbooks__panel column-4-span-3 flex flex--column justify--space-between"
      >
        <div
          className="RemoteLabbooks__icon-row"
        >
          <button
            className="RemoteLabbooks__cloud-download--exists"
            disabled={true}
          />
          <button
            className="RemoteLabbooks__cloud-delete"
            disabled={false}
            onClick={[Function]}
          />
        </div>
        <div
          className="RemoteLabbooks__text-row"
        >
          <div
            className="RemoteLabbooks__title-row"
          >
            <h6
              className="RemoteLabbooks__panel-title"
            >
              <span
                className={undefined}
              >
                <span
                  className=""
                  style={undefined}
                >
                  testing-activity-feed
                </span>
              </span>
            </h6>
          </div>
          <p
            className="RemoteLabbooks__owner"
          >
            Created by cbutler
          </p>
          <p
            className="RemoteLabbooks__description"
          >
            <span
              className={undefined}
            />
          </p>
        </div>
        <div
          className="Tooltip RemoteLabbooks__undefined"
          data-tooltip="undefined"
        />
      </div>
      <div
        className="RemoteLabbooks__panel column-4-span-3 flex flex--column justify--space-between"
      >
        <div
          className="RemoteLabbooks__icon-row"
        >
          <button
            className="RemoteLabbooks__cloud-download--exists"
            disabled={true}
          />
          <button
            className="RemoteLabbooks__cloud-delete"
            disabled={false}
            onClick={[Function]}
          />
        </div>
        <div
          className="RemoteLabbooks__text-row"
        >
          <div
            className="RemoteLabbooks__title-row"
          >
            <h6
              className="RemoteLabbooks__panel-title"
            >
              <span
                className={undefined}
              >
                <span
                  className=""
                  style={undefined}
                >
                  test-book
                </span>
              </span>
            </h6>
          </div>
          <p
            className="RemoteLabbooks__owner"
          >
            Created by cbutler
          </p>
          <p
            className="RemoteLabbooks__description"
          >
            <span
              className={undefined}
            />
          </p>
        </div>
        <div
          className="Tooltip RemoteLabbooks__undefined"
          data-tooltip="undefined"
        />
      </div>
      <div
        className="RemoteLabbooks__panel column-4-span-3 flex flex--column justify--space-between"
      >
        <div
          className="RemoteLabbooks__icon-row"
        >
          <button
            className="RemoteLabbooks__cloud-download--exists"
            disabled={true}
          />
          <button
            className="RemoteLabbooks__cloud-delete"
            disabled={false}
            onClick={[Function]}
          />
        </div>
        <div
          className="RemoteLabbooks__text-row"
        >
          <div
            className="RemoteLabbooks__title-row"
          >
            <h6
              className="RemoteLabbooks__panel-title"
            >
              <span
                className={undefined}
              >
                <span
                  className=""
                  style={undefined}
                >
                  temp-labook22
                </span>
              </span>
            </h6>
          </div>
          <p
            className="RemoteLabbooks__owner"
          >
            Created by cbutler
          </p>
          <p
            className="RemoteLabbooks__description"
          >
            <span
              className={undefined}
            />
          </p>
        </div>
        <div
          className="Tooltip RemoteLabbooks__undefined"
          data-tooltip="undefined"
        />
      </div>
      <div
        className="RemoteLabbooks__panel column-4-span-3 flex flex--column justify--space-between"
      >
        <div
          className="RemoteLabbooks__icon-row"
        >
          <button
            className="RemoteLabbooks__cloud-download--exists"
            disabled={true}
          />
          <button
            className="RemoteLabbooks__cloud-delete"
            disabled={false}
            onClick={[Function]}
          />
        </div>
        <div
          className="RemoteLabbooks__text-row"
        >
          <div
            className="RemoteLabbooks__title-row"
          >
            <h6
              className="RemoteLabbooks__panel-title"
            >
              <span
                className={undefined}
              >
                <span
                  className=""
                  style={undefined}
                >
                  temp
                </span>
              </span>
            </h6>
          </div>
          <p
            className="RemoteLabbooks__owner"
          >
            Created by cbutler
          </p>
          <p
            className="RemoteLabbooks__description"
          >
            <span
              className={undefined}
            />
          </p>
        </div>
        <div
          className="Tooltip RemoteLabbooks__undefined"
          data-tooltip="undefined"
        />
      </div>
      <div
        className="RemoteLabbooks__panel column-4-span-3 flex flex--column justify--space-between"
      >
        <div
          className="RemoteLabbooks__icon-row"
        >
          <button
            className="RemoteLabbooks__cloud-download"
            disabled={false}
            onClick={[Function]}
          />
          <button
            className="RemoteLabbooks__cloud-delete--disabled"
            disabled={false}
            onClick={[Function]}
          />
        </div>
        <div
          className="RemoteLabbooks__text-row"
        >
          <div
            className="RemoteLabbooks__title-row"
          >
            <h6
              className="RemoteLabbooks__panel-title"
            >
              <span
                className={undefined}
              >
                <span
                  className=""
                  style={undefined}
                >
                  t3
                </span>
              </span>
            </h6>
          </div>
          <p
            className="RemoteLabbooks__owner"
          >
            Created by bill2
          </p>
          <p
            className="RemoteLabbooks__description"
          >
            <span
              className={undefined}
            />
          </p>
        </div>
        <div
          className="Tooltip RemoteLabbooks__undefined"
          data-tooltip="undefined"
        />
      </div>
      <div
        className="RemoteLabbooks__panel column-4-span-3 flex flex--column justify--space-between"
      >
        <div
          className="RemoteLabbooks__icon-row"
        >
          <button
            className="RemoteLabbooks__cloud-download--exists"
            disabled={true}
          />
          <button
            className="RemoteLabbooks__cloud-delete"
            disabled={false}
            onClick={[Function]}
          />
        </div>
        <div
          className="RemoteLabbooks__text-row"
        >
          <div
            className="RemoteLabbooks__title-row"
          >
            <h6
              className="RemoteLabbooks__panel-title"
            >
              <span
                className={undefined}
              >
                <span
                  className=""
                  style={undefined}
                >
                  spade
                </span>
              </span>
            </h6>
          </div>
          <p
            className="RemoteLabbooks__owner"
          >
            Created by cbutler
          </p>
          <p
            className="RemoteLabbooks__description"
          >
            <span
              className={undefined}
            />
          </p>
        </div>
        <div
          className="Tooltip RemoteLabbooks__undefined"
          data-tooltip="undefined"
        />
      </div>
      <div
        className="RemoteLabbooks__panel column-4-span-3 flex flex--column justify--space-between"
      >
        <div
          className="RemoteLabbooks__icon-row"
        >
          <button
            className="RemoteLabbooks__cloud-download--exists"
            disabled={true}
          />
          <button
            className="RemoteLabbooks__cloud-delete--disabled"
            disabled={false}
            onClick={[Function]}
          />
        </div>
        <div
          className="RemoteLabbooks__text-row"
        >
          <div
            className="RemoteLabbooks__title-row"
          >
            <h6
              className="RemoteLabbooks__panel-title"
            >
              <span
                className={undefined}
              >
                <span
                  className=""
                  style={undefined}
                >
                  snappy
                </span>
              </span>
            </h6>
          </div>
          <p
            className="RemoteLabbooks__owner"
          >
            Created by gigantum-examples
          </p>
          <p
            className="RemoteLabbooks__description"
          >
            <span
              className={undefined}
            />
          </p>
        </div>
        <div
          className="Tooltip RemoteLabbooks__undefined"
          data-tooltip="undefined"
        />
      </div>
      <div
        className="RemoteLabbooks__panel column-4-span-3 flex flex--column justify--space-between"
      >
        <div
          className="RemoteLabbooks__icon-row"
        >
          <button
            className="RemoteLabbooks__cloud-download--exists"
            disabled={true}
          />
          <button
            className="RemoteLabbooks__cloud-delete"
            disabled={false}
            onClick={[Function]}
          />
        </div>
        <div
          className="RemoteLabbooks__text-row"
        >
          <div
            className="RemoteLabbooks__title-row"
          >
            <h6
              className="RemoteLabbooks__panel-title"
            >
              <span
                className={undefined}
              >
                <span
                  className=""
                  style={undefined}
                >
                  set-public-test-calum-true
                </span>
              </span>
            </h6>
          </div>
          <p
            className="RemoteLabbooks__owner"
          >
            Created by cbutler
          </p>
          <p
            className="RemoteLabbooks__description"
          >
            <span
              className={undefined}
            />
          </p>
        </div>
        <div
          className="Tooltip RemoteLabbooks__undefined"
          data-tooltip="undefined"
        />
      </div>
      <div
        className="RemoteLabbooks__panel column-4-span-3 flex flex--column justify--space-between"
      >
        <div
          className="RemoteLabbooks__icon-row"
        >
          <button
            className="RemoteLabbooks__cloud-download--exists"
            disabled={true}
          />
          <button
            className="RemoteLabbooks__cloud-delete"
            disabled={false}
            onClick={[Function]}
          />
        </div>
        <div
          className="RemoteLabbooks__text-row"
        >
          <div
            className="RemoteLabbooks__title-row"
          >
            <h6
              className="RemoteLabbooks__panel-title"
            >
              <span
                className={undefined}
              >
                <span
                  className=""
                  style={undefined}
                >
                  set-public-test-calum
                </span>
              </span>
            </h6>
          </div>
          <p
            className="RemoteLabbooks__owner"
          >
            Created by cbutler
          </p>
          <p
            className="RemoteLabbooks__description"
          >
            <span
              className={undefined}
            />
          </p>
        </div>
        <div
          className="Tooltip RemoteLabbooks__undefined"
          data-tooltip="undefined"
        />
      </div>
      <div
        className="RemoteLabbooks__panel column-4-span-3 flex flex--column justify--space-between"
      >
        <div
          className="RemoteLabbooks__icon-row"
        >
          <button
            className="RemoteLabbooks__cloud-download"
            disabled={false}
            onClick={[Function]}
          />
          <button
            className="RemoteLabbooks__cloud-delete--disabled"
            disabled={false}
            onClick={[Function]}
          />
        </div>
        <div
          className="RemoteLabbooks__text-row"
        >
          <div
            className="RemoteLabbooks__title-row"
          >
            <h6
              className="RemoteLabbooks__panel-title"
            >
              <span
                className={undefined}
              >
                <span
                  className=""
                  style={undefined}
                >
                  sdfdsfsdfsdf
                </span>
              </span>
            </h6>
          </div>
          <p
            className="RemoteLabbooks__owner"
          >
            Created by joel
          </p>
          <p
            className="RemoteLabbooks__description"
          >
            <span
              className={undefined}
            />
          </p>
        </div>
        <div
          className="Tooltip RemoteLabbooks__undefined"
          data-tooltip="undefined"
        />
      </div>
      <div
        className="RemoteLabbooks__panel column-4-span-3 flex flex--column justify--space-between"
      >
        <div
          className="RemoteLabbooks__icon-row"
        >
          <button
            className="RemoteLabbooks__cloud-download"
            disabled={false}
            onClick={[Function]}
          />
          <button
            className="RemoteLabbooks__cloud-delete--disabled"
            disabled={false}
            onClick={[Function]}
          />
        </div>
        <div
          className="RemoteLabbooks__text-row"
        >
          <div
            className="RemoteLabbooks__title-row"
          >
            <h6
              className="RemoteLabbooks__panel-title"
            >
              <span
                className={undefined}
              >
                <span
                  className=""
                  style={undefined}
                >
                  rootpy3
                </span>
              </span>
            </h6>
          </div>
          <p
            className="RemoteLabbooks__owner"
          >
            Created by gigantum-examples
          </p>
          <p
            className="RemoteLabbooks__description"
          >
            <span
              className={undefined}
            />
          </p>
        </div>
        <div
          className="Tooltip RemoteLabbooks__undefined"
          data-tooltip="undefined"
        />
      </div>
      <div
        className="RemoteLabbooks__panel column-4-span-3 flex flex--column justify--space-between"
      >
        <div
          className="RemoteLabbooks__icon-row"
        >
          <button
            className="RemoteLabbooks__cloud-download--exists"
            disabled={true}
          />
          <button
            className="RemoteLabbooks__cloud-delete--disabled"
            disabled={false}
            onClick={[Function]}
          />
        </div>
        <div
          className="RemoteLabbooks__text-row"
        >
          <div
            className="RemoteLabbooks__title-row"
          >
            <h6
              className="RemoteLabbooks__panel-title"
            >
              <span
                className={undefined}
              >
                <span
                  className=""
                  style={undefined}
                >
                  rootpy3
                </span>
              </span>
            </h6>
          </div>
          <p
            className="RemoteLabbooks__owner"
          >
            Created by randalg
          </p>
          <p
            className="RemoteLabbooks__description"
          >
            <span
              className={undefined}
            />
          </p>
        </div>
        <div
          className="Tooltip RemoteLabbooks__undefined"
          data-tooltip="undefined"
        />
      </div>
      <div
        className="RemoteLabbooks__panel column-4-span-3 flex flex--column justify--space-between"
      >
        <div
          className="RemoteLabbooks__icon-row"
        >
          <button
            className="RemoteLabbooks__cloud-download--exists"
            disabled={true}
          />
          <button
            className="RemoteLabbooks__cloud-delete"
            disabled={false}
            onClick={[Function]}
          />
        </div>
        <div
          className="RemoteLabbooks__text-row"
        >
          <div
            className="RemoteLabbooks__title-row"
          >
            <h6
              className="RemoteLabbooks__panel-title"
            >
              <span
                className={undefined}
              >
                <span
                  className=""
                  style={undefined}
                >
                  random-picture-generator
                </span>
              </span>
            </h6>
          </div>
          <p
            className="RemoteLabbooks__owner"
          >
            Created by cbutler
          </p>
          <p
            className="RemoteLabbooks__description"
          >
            <span
              className={undefined}
            />
          </p>
        </div>
        <div
          className="Tooltip RemoteLabbooks__undefined"
          data-tooltip="undefined"
        />
      </div>
      <div
        className="RemoteLabbooks__panel column-4-span-3 flex flex--column justify--space-between"
      >
        <div
          className="RemoteLabbooks__icon-row"
        >
          <button
            className="RemoteLabbooks__cloud-download"
            disabled={false}
            onClick={[Function]}
          />
          <button
            className="RemoteLabbooks__cloud-delete--disabled"
            disabled={false}
            onClick={[Function]}
          />
        </div>
        <div
          className="RemoteLabbooks__text-row"
        >
          <div
            className="RemoteLabbooks__title-row"
          >
            <h6
              className="RemoteLabbooks__panel-title"
            >
              <span
                className={undefined}
              >
                <span
                  className=""
                  style={undefined}
                >
                  python-data-science-handbook
                </span>
              </span>
            </h6>
          </div>
          <p
            className="RemoteLabbooks__owner"
          >
            Created by dmk
          </p>
          <p
            className="RemoteLabbooks__description"
          >
            <span
              className={undefined}
            />
          </p>
        </div>
        <div
          className="Tooltip RemoteLabbooks__undefined"
          data-tooltip="undefined"
        />
      </div>
      <div
        className="RemoteLabbooks__panel column-4-span-3 flex flex--column justify--space-between"
      >
        <div
          className="RemoteLabbooks__icon-row"
        >
          <button
            className="RemoteLabbooks__cloud-download--exists"
            disabled={true}
          />
          <button
            className="RemoteLabbooks__cloud-delete"
            disabled={false}
            onClick={[Function]}
          />
        </div>
        <div
          className="RemoteLabbooks__text-row"
        >
          <div
            className="RemoteLabbooks__title-row"
          >
            <h6
              className="RemoteLabbooks__panel-title"
            >
              <span
                className={undefined}
              >
                <span
                  className=""
                  style={undefined}
                >
                  pulish-temp4
                </span>
              </span>
            </h6>
          </div>
          <p
            className="RemoteLabbooks__owner"
          >
            Created by cbutler
          </p>
          <p
            className="RemoteLabbooks__description"
          >
            <span
              className={undefined}
            />
          </p>
        </div>
        <div
          className="Tooltip RemoteLabbooks__undefined"
          data-tooltip="undefined"
        />
      </div>
      <div
        className="Labbook__loader-hidden"
      >
        <div />
      </div>
      <div
        className="Labbook__loader-hidden"
      >
        <div />
      </div>
      <div
        className="Labbook__loader-hidden"
      >
        <div />
      </div>
      <div
        className="Labbook__loader-hidden"
      >
        <div />
      </div>
      <div
        className="Labbook__loader-hidden"
      >
        <div />
      </div>
    </div>
  </div>
</div>
`;
